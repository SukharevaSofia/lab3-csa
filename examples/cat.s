_LOOP:
  in r0
  out r0
  jump _LOOP
  hlt
