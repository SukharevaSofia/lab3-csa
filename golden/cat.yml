source: |-
  _LOOP:
    in r0
    out r0
    jump _LOOP
    hlt

stdin: |-
  eqwueyuqwiyeuqwyeuqywuieyqwuieyuqwyeuqwyuieyqwui

target: |-
  {"program": [{"instr": "in", "args": [0]}, {"instr": "out", "args": [0]}, {"instr": "jump", "args": [0]}, {"instr": "hlt", "args": []}], "data": []}
stdout: |-
  eqwueyuqwiyeuqwyeuqywuieyqwuieyuqwyeuqwyuieyqwui

logs: ./golden/logs/cat.log
